src/realm/util/timestamp_logger.hpp

```diff
@@ -14,7 +14,7 @@ class TimestampStderrLogger : public Logger {
     using Precision = TimestampFormatter::Precision;
     using Config = TimestampFormatter::Config;
 
-    explicit TimestampStderrLogger(Config = {}, Level = Level::info);
+    explicit TimestampStderrLogger(Config = {}, Level = default_log_level);
```

Comment:
I know that default_log_level is available as a member here because we inherit from Logger, but could you make this explicit like `Logger::default_log_level` anyways?